packages/compartment-mapper/package.json

```diff
@@ -28,8 +28,14 @@
       "default": "./index.js"
     },
     "./import.js": "./import.js",
+    "./import-lite.js": "./import-lite.js",
```

Comment:
(suggestion/observation/trolling) there's obviously a convention at play here, but I'd omit the `.js` from the keys

Reply:
I very much want to. It’s not just convention, though. This makes the Compartment Mapper continue to work when used on some older foundations like `node -r esm` that do not read the `"exports"` directive, so we can’t use it for aliases, just to limit exposure. We have one lingering integration test that needs to be replaced before we can drop this pattern.

Reply:
Didn't realize that `esm` was a target